setup.py

```diff
@@ -27,27 +27,30 @@
     readme = f.read()
 
 kerberos_require = ["requests_kerberos"]
-gssapi_require = [""
-                  "requests_gssapi",
-                  # PyPy compatibility issue https://github.com/jborean93/pykrb5/issues/49
-                  "krb5 == 0.5.1"]
+gssapi_require = [
+    "" "requests_gssapi",
+    # PyPy compatibility issue https://github.com/jborean93/pykrb5/issues/49
+    "krb5 == 0.5.1",
+]
 sqlalchemy_require = ["sqlalchemy >= 1.3"]
 external_authentication_token_cache_require = ["keyring"]
 
 # We don't add localstorage_require to all_require as users must explicitly opt in to use keyring.
 all_require = kerberos_require + sqlalchemy_require
 
-tests_require = all_require + gssapi_require + [
-    # httpretty >= 1.1 duplicates requests in `httpretty.latest_requests`
-    # https://github.com/gabrielfalcao/HTTPretty/issues/425
-    "httpretty < 1.1",
-    "pytest",
-    "pytest-runner",
-    "pre-commit",
-    "black",
-    "isort",
```

Comment:
Both `isort` and `black` are here but were never enforced. Neither were mentioned in the development documentation as well.

Reply:
might want to look at https://github.com/trinodb/trino-python-client/pull/227 for context.

The TL;DR version was:

- While standard code format is nice black often makes changes to the detriment of readability
- If we ever do this we need to figure out a way to be history preserving (probably via blame-ignore config - https://github.com/jvanzyl/provisio/commit/a4cdb8dfce376eb8482f78681596f264705b0f86)
- black and isort running in pre-commit makes iteration slower sometimes since you are prevented from committing or running ci just because of style issues